### PR TITLE
Add non-browser feedback about unsigned bundles.

### PR DESCRIPTION
**Pull request by jyasskin**
@dauwhe, am I representing your position well here? Is there a better description of your opinion on how we make browsers navigate to unsigned books?

### CODE REVIEW

**Comment by KenjiBaheux** [comment]
Would this be the right section to add use cases such as the ones that Dan York had in mind, e.g. making documentation for internet infra grant available for offline consumption (see https://www.internetsociety.org/grants/chapterthon/2019/)?

**Comment by jyasskin** [comment]
@KenjiBaheux It would, but I would appreciate if you or @danyork could suggest some text.

File: explainers/navigation-to-unsigned-bundles.md [hunk 1]
```diff
@@ -375,6 +375,38 @@ full URLs needed by web bundles.
...
+    packages be used for ad-hoc content distribution? I email you a package; you
+    double-click and it opens in your browser. How can web packaging become as
+    simple as PDF?"
+  * ["From an archival standpoint, the potential for Web content to be provided
+    in a self-contained form was viewed
+    positively."](https://tools.ietf.org/html/draft-iab-escape-report-00#section-2.3.2),
+    but archivists were worried that content producers might not package their
+    content enough to make it fully effective. They also thought packages might
+    improve the "situation for archival replay".
+* Computer vendors: Interest in using bundles to represent pre-installed
```
Comment (KenjiBaheux) [inline explainers/navigation-to-unsigned-bundles.md:408]: * [Dan York of Internet Society](https://www.iab.org/wp-content/IAB-uploads/2019/06/dan-york.pdf) offered use cases for packaging web content focused on availability in low bandwidth / high latency environments. More specifically, part of the ISOC's mission involves reaching people who only have have limited connectivity to the Internet, e.g. WiFi at a few local places that people visit to download content that they want to read or watch later. The ability to package web content would be useful in such context. The ISOC is also working with communities where the electricity is unreliable, which negatively impacts Internet access as a result. Here again, these communities could benefit from the ability to download a package of content, either for learning or reading even in the absence of connectivity.

Comment (KenjiBaheux) [reply explainers/navigation-to-unsigned-bundles.md:408]: Sorry for the delay, I think this captures what Dan wrote in his position paper but would welcome any feedback or direct edits.

Comment (jyasskin) [reply explainers/navigation-to-unsigned-bundles.md:785]: Thanks for the text. After a long delay, it's merged. Everything look good now?

Comment (danyork) [reply explainers/navigation-to-unsigned-bundles.md:785]: @jyasskin and @KenjiBaheux - Thanks for the summary and my apologies for a delay in replying. The text looks good overall, but I'd suggest two changes:

- we like to spell out "Internet Society" versus using "ISOC" (two many acronym conflicts in the larger world)
- I would add in a mention of satellite connectivity.

Perhaps the text would be better as:

Dan York of Internet Society offered use cases for packaging web content focused on availability in low bandwidth / high latency environments. More specifically, part of the Internet Society's work involves reaching people who only have have limited connectivity to the Internet, e.g. WiFi at a few local places that people visit to download content that they want to read or watch later. The ability to package web content would be useful in such context. The Internet Society is also working with communities where the electricity is unreliable, which negatively impacts Internet access as a result. There are also cases where communities only have periodic connectivity via satellite connections. Here again, these communities could benefit from the ability to download a package of content, either for learning or reading even in the absence of connectivity.

Comment (jyasskin) [reply explainers/navigation-to-unsigned-bundles.md:787]: Thanks! I've taken that wording.

**Pull request merged**